src/plugins/indicator_libmessaging.py

```diff
@@ -1,14 +1,18 @@
 # -*- coding: utf-8 -*-
+"""
+src/plugins/indicator_libmessaging.py
+=====================================
+"""
+from pybitmessage.bitmessageqt.utils import str_broadcast_subscribers
+from pybitmessage.tr import _translate
 
 import gi
-gi.require_version('MessagingMenu', '1.0')  # noqa:E402
+gi.require_version('MessagingMenu', '1.0')  # ..noqa:E402
```

Comment:
This should just remain as it was, no `..`

Reply:
Okay.